src/ethereum_clis/ethereum_cli.py

```diff
@@ -38,24 +38,23 @@ class EthereumCLI:
 
     registered_tools: List[Type[Any]] = []
     default_tool: Optional[Type[Any]] = None
-    binary: Path
+    binary: Optional[Path]
     default_binary: Path
     detect_binary_pattern: Pattern
     version_flag: str = "-v"
     cached_version: Optional[str] = None
 
     def __init__(self, *, binary: Optional[Path] = None):
         """Abstract initialization method that all subclasses must implement."""
+        # If an evm_bin is not specified, use the eels native t8n
         if binary is None:
-            binary = self.default_binary
-        else:
-            # improve behavior of which by resolving the path: ~/relative paths don't work
-            resolved_path = Path(os.path.expanduser(binary)).resolve()
-            if resolved_path.exists():
-                binary = resolved_path
+            return
+
+        # improve behavior of which by resolving the path: ~/relative paths don't work
+        resolved_path = Path(os.path.expanduser(binary)).resolve()
+        if resolved_path.exists():
+            binary = resolved_path
         binary = shutil.which(binary)  # type: ignore
-        if not binary:
-            raise CLINotFoundInPathError(binary=binary)
```

Comment:
This looks like it'll give bad ux/unexpected errors for non-eels tools that don't get found in the PATH. Would need to test.